open_spiel/algorithms/external_sampling_mccfr.cc

```diff
@@ -26,6 +26,23 @@
 namespace open_spiel {
 namespace algorithms {
 
+// Serialization of the MCCFR solver is in agreement with de/serialization of
```

Comment:
Why copy these again here? Just put them in cfr.h and re-use them?

Reply:
Done.